services/movie_session.py

```diff
@@ -0,0 +1,47 @@
+import datetime
+
+from django.db.models import QuerySet
+
+from db.models import MovieSession, CinemaHall, Movie
+
+
+def create_movie_session(
+    movie_show_time: datetime, movie_id: int, cinema_hall_id: int
+) -> None:
+    MovieSession.objects.create(
+        show_time=movie_show_time,
+        cinema_hall=CinemaHall.objects.get(id=cinema_hall_id),
+        movie=Movie.objects.get(id=movie_id),
+    )
+
+
+def get_movies_sessions(session_date: str = None) -> QuerySet:
+    if session_date is None:
+        return MovieSession.objects.all()
+    return MovieSession.objects.filter(
+        show_time__date=datetime.datetime.strptime(session_date, "%Y-%m-%d")
+    )
+
+
+def get_movie_session_by_id(movie_session_id: int) -> MovieSession:
+    return MovieSession.objects.get(id=movie_session_id)
+
+
+def update_movie_session(
+    session_id: int,
+    show_time: datetime = None,
+    movie_id: int = None,
+    cinema_hall_id: int = None,
+) -> None:
+    movie_session = MovieSession.objects.get(id=session_id)
+    if show_time is not None:
+        movie_session.show_time = show_time
+    if movie_id is not None:
+        movie_session.movie = Movie.objects.get(id=movie_id)
+    if cinema_hall_id is not None:
+        movie_session.cinema_hall = CinemaHall.objects.get(id=cinema_hall_id)
```

Comment:
How do you want to change an object of movie without getting an object than?

Reply:
you have `movie_session = MovieSession.objects.get(id=session_id)` so this is object,
and you have `movie_id` its `id` , is there a way to update an object with `id `only, without querying to DB for movie object itself?

Reply:
![image](https://github.com/mate-academy/py-movie-moviesession-cinemahall/assets/106102798/3069faff-c7d7-4dda-b7fe-7a7e4ea58ed9)
![image](https://github.com/mate-academy/py-movie-moviesession-cinemahall/assets/106102798/c47564c2-ec79-4402-a34a-b4875bc3bebb)
Yes, we do need an instance, we can't update only ID because Django doesn't understand id changes